data/transit/route/bus.json

```diff
@@ -2586,12 +2586,14 @@
       }
     },
     {
-      "displayName": "DE_CO_BUS",
+      "displayName": "Verkehrsgemeinschaft Coburg",
       "id": "decobus-3b9544",
       "locationSet": {"include": ["de"]},
       "tags": {
-        "network": "DE_CO_BUS",
-        "operator": "DE_CO_BUS",
+        "network": "Verkehrsgemeinschaft Coburg",
+        "network": "VGC",
```

Comment:
I think you mean:

```suggestion
        "network": "Verkehrsgemeinschaft Coburg",
        "network:short": "VGC",
```

Reply:
Thanks for the review! As there is no common approach on naming networks in Germany (abbreviated or not?) I decided to close this pull request in lieu of another one with names left untouched but additional wikidata, wikipedia and some minor changes.